checker/src/main/java/org/checkerframework/checker/nullness/CollectionToArrayHeuristics.java

```diff
@@ -67,11 +76,12 @@ public CollectionToArrayHeuristics(
     public void handle(MethodInvocationTree tree, AnnotatedExecutableType method) {
         if (TreeUtils.isMethodInvocation(tree, collectionToArrayE, processingEnv)) {
             assert !tree.getArguments().isEmpty() : tree;
-            Tree argument = tree.getArguments().get(0);
+            ExpressionTree argument = tree.getArguments().get(0);
             boolean argIsArrayCreation =
```

Comment:
This lost a few changes I made in https://github.com/typetools/checker-framework/commit/f33adcdeaea4eb98105e104dacd20fff6627452c

Reply:
I successfully cherry-picked that changeset.